macro/macroutils.py

```diff
@@ -175,7 +175,7 @@ def load_stack_release(release_name, stack_name):
         if release_name in ['boxturtle', 'cturtle', 'diamondback']:
             usock = open('/var/www/www.ros.org/distros/%s.rosdistro'%release_name)
         else:
-            usock = urllib2.urlopen('http://ros-dry-releases.googlecode.com/svn/trunk/distros/%s.rosdistro'%release_name)
+            usock = urllib2.urlopen('http://svn.code.sf.net/p/ros-dry-releases/code/trunk/distros/%s.rosdistro'%release_name)
```

Comment:
It looks like this SourceForge URL is only good for electric, fuerte, groovy, and hydro (although hydro is basically empty). What's the idea for the still-supported distros?

Reply:
This is only for legacy compatibility, we don't do rosbuild releases since then. (`dry` == rosbuild) anything since then is using bloom and catkin.

Reply:
Makes sense, thanks.